examples/arkode/CXX_parallel/ark_brusselator1D_task_local_nls.USE_SERIAL_NVEC_--monitor_--explicit_--tf_1.out

```diff
@@ -25,47 +25,47 @@ Output directory: .
+       0.125000     1.093430     3.517723     3.499974
+       0.150000     1.101013     3.507895     3.499995
+       0.175000     1.108944     3.497559     3.499976
+       0.200000     1.117237     3.486697     3.499973
+       0.225000     1.125909     3.475285     3.499988
+       0.250000     1.134977     3.463305     3.499976
+       0.275000     1.144458     3.450734     3.499979
+       0.300000     1.154369     3.437549     3.499986
+       0.325000     1.164729     3.423731     3.499971
+       0.350000     1.175555     3.409255     3.499981
+       0.375000     1.186865     3.394103     3.499990
+       0.400000     1.198677     3.378251     3.499975
+       0.425000     1.211006     3.361681     3.499979
+       0.450000     1.223870     3.344373     3.499978
+       0.475000     1.237283     3.326310     3.499974
+       0.500000     1.251256     3.307478     3.499992
+       0.525000     1.265800     3.287863     3.499965
+       0.550000     1.280922     3.267456     3.499971
+       0.575000     1.296624     3.246252     3.499986
+       0.600000     1.312905     3.224250     3.499972
+       0.625000     1.329757     3.201455     3.499977
        0.650000     1.347164     3.177875     3.499981
        0.675000     1.365106     3.153529     3.499968
-       0.700000     1.383550     3.128441     3.499979
-       0.725000     1.402455     3.102644     3.499967
-       0.750000     1.421770     3.076177     3.499974
-       0.775000     1.441431     3.049092     3.499973
-       0.800000     1.461363     3.021446     3.499971
-       0.825000     1.481479     2.993308     3.499982
-       0.850000     1.501680     2.964753     3.499970
-       0.875000     1.521856     2.935864     3.499985
-       0.900000     1.541887     2.906731     3.499972
+       0.700000     1.383550     3.128441     3.499991
+       0.725000     1.402455     3.102644     3.499983
+       0.750000     1.421770     3.076177     3.499965
+       0.775000     1.441431     3.049092     3.499978
+       0.800000     1.461363     3.021446     3.499979
+       0.825000     1.481479     2.993308     3.499971
+       0.850000     1.501680     2.964753     3.499979
+       0.875000     1.521856     2.935864     3.499961
+       0.900000     1.541887     2.906731     3.499979
        0.925000     1.561648     2.877449     3.499982
-       0.950000     1.581005     2.848117     3.499976
-       0.975000     1.599823     2.818833     3.499969
-       1.000000     1.617967     2.789700     3.499977
+       0.950000     1.581005     2.848117     3.499966
+       0.975000     1.599823     2.818833     3.499973
+       1.000000     1.617967     2.789700     3.499975
 
 Final Solver Statistics (for processor 0):
-   Internal solver steps = 79612 (attempted = 79835)
-   Total RHS evals:  Fe = 239508   Total number of error test failures = 223
+   Internal solver steps = 78730 (attempted = 110699)
+   Total RHS evals:  Fe = 332100   Total number of error test failures = 31969
```

Comment:
Steven confirmed this increase in the number of error test failures is fixed by the updated controller defaults in PR #565

Reply:
Fixed by commit 40dd2bb